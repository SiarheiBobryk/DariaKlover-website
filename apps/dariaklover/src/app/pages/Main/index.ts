export { default } from './Main';

export { default as mainMetaData } from './mainMetaData';
export * from './mainMetaData';
