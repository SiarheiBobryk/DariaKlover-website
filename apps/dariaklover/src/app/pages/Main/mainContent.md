Всем привет!

Я Дарья Кловер - дипломированный health-коуч, специалист по здоровому образу жизни и немедикаментозному оздоровлению.

Помогаю своим клиентам вести здоровый образ жизни и предотвратить появление хронических заболеваний, а также повысить качество жизни, улучшить самочувствие, делюсь рекомендациями для восстановления и поддержания здоровья.

Разбираю запрос клиента с разных сторон, что помогает увидеть полную картину конкретной ситуации и ее причины. Использую различные подходы и инструменты, чтобы помочь проработать эти причины и прийти к оптимальному решению.

Одна из моих главных задач - осознание клиентом того, **что, как и почему нужно делать**, чтобы достичь желаемого результата.

_Примеры запросов_, с которыми я работаю:

-   **хочу, знаю, но не делаю**
-   **привести себя в форму**
-   **снизить/набрать вес**
-   **вернуть энергию**
-   **найти мотивацию**
-   **преодолеть страх**
-   **внедрить привычки**
-   **достичь цели**
-   **наладить сон**
-   **сбалансированно питаться**
-   **организовать свое время**
-   **облегчить адаптацию на новом месте**
-   **нет понимания, что делать дальше**

Если ваш запрос отличается от приведенных выше, не переживайте. Мы можем обсудить это с вами на установочной встрече и решить, чем я смогу вам помочь или порекомендую другого специалиста.
