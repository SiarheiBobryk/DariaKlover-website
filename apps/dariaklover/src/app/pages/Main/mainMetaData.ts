import { PageMetaData } from '@dariaklover/types';

const mainMetaData: PageMetaData = {
  title: 'Главная страница',
  heading: 'Кто я и что делаю',
  path: '/',
};

export default mainMetaData;
