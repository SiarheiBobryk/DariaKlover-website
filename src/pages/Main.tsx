import * as React from 'react';

function Main() {
  return <span>Главная</span>;
}

export default Main;
