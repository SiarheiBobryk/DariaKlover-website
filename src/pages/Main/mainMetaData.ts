export interface MainMetaData {
  title: string;
  heading: string;
  path: string;
}

const mainMetaData: MainMetaData = {
  title: 'Главная страница',
  heading: 'Кто я и что я делаю',
  path: '/',
};

export default mainMetaData;
