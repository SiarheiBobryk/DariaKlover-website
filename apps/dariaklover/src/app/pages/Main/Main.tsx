import * as React from 'react';
import { Helmet } from 'react-helmet';

import AvatarMui from '@mui/material/Avatar';
import { styled } from '@mui/material/styles';

import Heading from '../../components/Heading';
import Markdown from '../../components/Markdown';

import mainContent from './mainContent.md';
import mainMetaData from './mainMetaData';

const { NX_IMAGES_PATH = '' } = process.env;

export const AvatarContainer = styled('div')(function styleAvatarContainer() {
  return {
    display: 'flex',
    justifyContent: 'center',
  };
});

export const Avatar = styled(AvatarMui)(function styleAvatar({ theme }) {
  return {
    width: theme.spacing(35),
    height: theme.spacing(35),
    margin: theme.spacing(1),
  };
});

function Main() {
  return (
    <>
      <Helmet>
        <title>{mainMetaData.title}</title>
      </Helmet>
      <Heading>{mainMetaData.heading}</Heading>
      <AvatarContainer>
        <Avatar alt="Дарья Кловер" src={`${NX_IMAGES_PATH}/photo-s.jpg`} />
      </AvatarContainer>
      <Markdown>{mainContent}</Markdown>
    </>
  );
}

export default Main;
