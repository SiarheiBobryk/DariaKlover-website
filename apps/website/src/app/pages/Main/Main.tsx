import * as React from 'react';
import { Helmet } from 'react-helmet';

import MainMdx from './main.mdx';
import mainMetaData from './mainMetaData';

function Main() {
  return (
    <>
      <Helmet>
        <title>{mainMetaData.title}</title>
      </Helmet>
      <MainMdx />
    </>
  );
}

export default Main;
