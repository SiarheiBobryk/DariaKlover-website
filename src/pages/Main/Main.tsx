import * as React from 'react';
import { Helmet } from 'react-helmet';

import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';
import { styled } from '@mui/material/styles';

import Heading from '../../components/Heading';
import Paragraph from '../../components/Paragraph';
import LinkBlank from '../../components/LinkBlank';
import mainMetaData from './mainMetaData';

export const Strong = styled('strong')(() => {
  return {
    textDecoration: 'underline',
  };
});

function Main() {
  return (
    <>
      <Helmet>
        <title>{mainMetaData.title}</title>
      </Helmet>
      <Heading>{mainMetaData.heading}</Heading>
      <Paragraph>–í—Å–µ–º –ø—Ä–∏–≤–µ—Ç, —è - –î–∞—Ä—å—è –ö–ª–æ–≤–µ—Ä!</Paragraph>
      <Paragraph>
        –Ø –ø—Ä–æ—Ö–æ–∂—É –æ–±—É—á–µ–Ω–∏–µ –≤{' '}
        <LinkBlank href="https://uom.education/university" rel="noopener">
          –£–Ω–∏–≤–µ—Ä—Å–∏—Ç–µ—Ç–µ –æ–±—Ä–∞–∑–æ–≤–∞—Ç–µ–ª—å–Ω–æ–π –º–µ–¥–∏—Ü–∏–Ω—ã (–£–û–ú)
        </LinkBlank>{' '}
        –∏ <i>PreventAge LifeStyle School</i> –Ω–∞ –∏–Ω—Ç–µ–≥—Ä–∞—Ç–∏–≤–Ω–æ–≥–æ health-–∫–æ—É—á–∞ - —Å–ø–µ—Ü–∏–∞–ª–∏—Å—Ç–∞ –ø–æ –º–æ–¥–∏—Ñ–∏–∫–∞—Ü–∏–∏ –æ–±—Ä–∞–∑–∞ –∂–∏–∑–Ω–∏ –∏
        –Ω–µ–º–µ–¥–∏–∫–∞–º–µ–Ω—Ç–æ–∑–Ω–æ–º—É –æ–∑–¥–æ—Ä–æ–≤–ª–µ–Ω–∏—é.
      </Paragraph>
      <Paragraph>
        –Ø –ø–æ–º–æ–≥–∞—é —Å–≤–æ–∏–º –∫–ª–∏–µ–Ω—Ç–∞–º –≤–µ—Å—Ç–∏ –∑–¥–æ—Ä–æ–≤—ã–π –æ–±—Ä–∞–∑ –∂–∏–∑–Ω–∏ –∏ –ø—Ä–µ–¥–æ—Ç–≤—Ä–∞—Ç–∏—Ç—å –ø–æ—è–≤–ª–µ–Ω–∏–µ —Ö—Ä–æ–Ω–∏—á–µ—Å–∫–∏—Ö –∑–∞–±–æ–ª–µ–≤–∞–Ω–∏–π, –∞ —Ç–∞–∫–∂–µ
        –ø–æ–≤—ã—Å–∏—Ç—å –∫–∞—á–µ—Å—Ç–≤–æ –∂–∏–∑–Ω–∏, —É–ª—É—á—à–∏—Ç—å —Å–∞–º–æ—á—É–≤—Å—Ç–≤–∏–µ, –¥–µ–ª—é—Å—å —Ä–µ–∫–æ–º–µ–Ω–¥–∞—Ü–∏—è–º–∏ –¥–ª—è –≤–æ—Å—Å—Ç–∞–Ω–æ–≤–ª–µ–Ω–∏—è –∏ –ø–æ–¥–¥–µ—Ä–∂–∞–Ω–∏—è –∑–¥–æ—Ä–æ–≤—å—è.
      </Paragraph>

      <Paragraph>
        –Ø —Ä–∞–∑–±–∏—Ä–∞—é –∑–∞–ø—Ä–æ—Å –∫–ª–∏–µ–Ω—Ç–∞ —Å —Ä–∞–∑–Ω—ã—Ö —Å—Ç–æ—Ä–æ–Ω, —á—Ç–æ –ø–æ–º–æ–≥–∞–µ—Ç —É–≤–∏–¥–µ—Ç—å –ø–æ–ª–Ω—É—é –∫–∞—Ä—Ç–∏–Ω—É –∫–æ–Ω–∫—Ä–µ—Ç–Ω–æ–π —Å–∏—Ç—É–∞—Ü–∏–∏ –∏ –µ–µ –ø—Ä–∏—á–∏–Ω—ã.
        –Ø –∏—Å–ø–æ–ª—å–∑—É—é —Ä–∞–∑–ª–∏—á–Ω—ã–µ –ø–æ–¥—Ö–æ–¥—ã –∏ –∏–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç—ã, —á—Ç–æ–±—ã –ø–æ–º–æ—á—å –ø—Ä–æ—Ä–∞–±–æ—Ç–∞—Ç—å —ç—Ç–∏ –ø—Ä–∏—á–∏–Ω—ã –∏ –ø—Ä–∏–¥—Ç–∏ –∫ —Ä–µ—à–µ–Ω–∏—é –∑–∞–ø—Ä–æ—Å–∞
        –∫–ª–∏–µ–Ω—Ç–∞.
      </Paragraph>

      {/* TODO: The better (React) alternative for `strong` here? */}
      <Paragraph>
        –û–¥–Ω–∞ –∏–∑ –º–æ–∏—Ö –≥–ª–∞–≤–Ω—ã—Ö –∑–∞–¥–∞—á - –æ—Å–æ–∑–Ω–∞–Ω–∏–µ –∫–ª–∏–µ–Ω—Ç–æ–º —Ç–æ–≥–æ, <Strong>—á—Ç–æ, –∫–∞–∫ –∏ –ø–æ—á–µ–º—É –Ω—É–∂–Ω–æ –¥–µ–ª–∞—Ç—å</Strong>, —á—Ç–æ–±—ã
        –¥–æ—Å—Ç–∏—á—å –∂–µ–ª–∞–µ–º–æ–≥–æ —Ä–µ–∑—É–ª—å—Ç–∞—Ç–∞.
      </Paragraph>
      <Alert sx={{ my: 1 }} severity="info">
        <AlertTitle>–í–µ–¥—É—Ç—Å—è —Ä–∞–±–æ—Ç—ã</AlertTitle>–í —Å–≤—è–∑–∏ —Å —Ç–µ–º, —á—Ç–æ –Ω–∞ —Å–∞–π—Ç–µ –≤–µ–¥—É—Ç—Å—è –∞–∫—Ç–∏–≤–Ω—ã–µ —Ä–∞–±–æ—Ç—ã, —á—Ç–æ-—Ç–æ –º–æ–∂–µ—Ç
        –æ—Ç—Å—É—Ç—Å—Ç–≤–æ–≤–∞—Ç—å –∏–ª–∏ –Ω–µ —Ä–∞–±–æ—Ç–∞—Ç—å. –°–ø–∞—Å–∏–±–æ –∑–∞ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
      </Alert>
    </>
  );
}

export default Main;
