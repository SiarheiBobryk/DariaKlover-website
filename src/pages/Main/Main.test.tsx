import * as React from 'react';
import { Helmet, HelmetPropsToState } from 'react-helmet';

import { axe, toHaveNoViolations } from 'jest-axe';
import { render, RenderResult } from '@testing-library/react';

import { typographyClasses } from '@mui/material/Typography';

import Main from './Main';
import mainMetaData from './mainMetaData';

expect.extend(toHaveNoViolations);

describe('<Main />', () => {
  describe('a11y', () => {
    it('should match a11y standards', async () => {
      const { container }: RenderResult = render(<Main />);

      const results = await axe(container);
      expect(results).toHaveNoViolations();
    });
  });

  describe('Elements inspection', () => {
    it('should have the right title HTML element', () => {
      render(<Main />);

      const { title }: HelmetPropsToState = Helmet.peek();
      expect(title).toBe(mainMetaData.title);
    });

    it('should have the `h2` HTML element as heading', () => {
      render(<Main />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toBeInTheDocument();
    });

    it('should have the CSS classes from `h5` Typography variant by default', () => {
      render(<Main />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toHaveClass(typographyClasses.h5);
    });

    it('should have the right `strong` element', () => {
      render(<Main />);

      const strong: HTMLElement | null = document.querySelector('strong');
      expect(strong).toHaveTextContent('что, как и почему нужно делать');
    });
  });
});
