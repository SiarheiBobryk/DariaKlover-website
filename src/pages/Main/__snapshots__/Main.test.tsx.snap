// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> snapshotting should render correctly 1`] = `
Array [
  <h2
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
  >
    –ö—Ç–æ —è –∏ —á—Ç–æ —è –¥–µ–ª–∞—é
  </h2>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –í—Å–µ–º –ø—Ä–∏–≤–µ—Ç, —è - –î–∞—Ä—å—è –ö–ª–æ–≤–µ—Ä!
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –Ø –ø—Ä–æ—Ö–æ–∂—É –æ–±—É—á–µ–Ω–∏–µ –≤
     
    <a
      className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
      href="https://uom.education/university"
      onBlur={[Function]}
      onFocus={[Function]}
      rel="noopener"
      target="_blank"
    >
      –£–Ω–∏–≤–µ—Ä—Å–∏—Ç–µ—Ç–µ –æ–±—Ä–∞–∑–æ–≤–∞—Ç–µ–ª—å–Ω–æ–π –º–µ–¥–∏—Ü–∏–Ω—ã (–£–û–ú)
       
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
        data-testid="OpenInNewIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
        />
      </svg>
    </a>
     
    –∏ 
    <i>
      PreventAge LifeStyle School
    </i>
     –Ω–∞ –∏–Ω—Ç–µ–≥—Ä–∞—Ç–∏–≤–Ω–æ–≥–æ health-–∫–æ—É—á–∞ - —Å–ø–µ—Ü–∏–∞–ª–∏—Å—Ç–∞ –ø–æ –º–æ–¥–∏—Ñ–∏–∫–∞—Ü–∏–∏ –æ–±—Ä–∞–∑–∞ –∂–∏–∑–Ω–∏ –∏ –Ω–µ–º–µ–¥–∏–∫–∞–º–µ–Ω—Ç–æ–∑–Ω–æ–º—É –æ–∑–¥–æ—Ä–æ–≤–ª–µ–Ω–∏—é.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –Ø –ø–æ–º–æ–≥–∞—é —Å–≤–æ–∏–º –∫–ª–∏–µ–Ω—Ç–∞–º –≤–µ—Å—Ç–∏ –∑–¥–æ—Ä–æ–≤—ã–π –æ–±—Ä–∞–∑ –∂–∏–∑–Ω–∏ –∏ –ø—Ä–µ–¥–æ—Ç–≤—Ä–∞—Ç–∏—Ç—å –ø–æ—è–≤–ª–µ–Ω–∏–µ —Ö—Ä–æ–Ω–∏—á–µ—Å–∫–∏—Ö –∑–∞–±–æ–ª–µ–≤–∞–Ω–∏–π, –∞ —Ç–∞–∫–∂–µ –ø–æ–≤—ã—Å–∏—Ç—å –∫–∞—á–µ—Å—Ç–≤–æ –∂–∏–∑–Ω–∏, —É–ª—É—á—à–∏—Ç—å —Å–∞–º–æ—á—É–≤—Å—Ç–≤–∏–µ, –¥–µ–ª—é—Å—å —Ä–µ–∫–æ–º–µ–Ω–¥–∞—Ü–∏—è–º–∏ –¥–ª—è –≤–æ—Å—Å—Ç–∞–Ω–æ–≤–ª–µ–Ω–∏—è –∏ –ø–æ–¥–¥–µ—Ä–∂–∞–Ω–∏—è –∑–¥–æ—Ä–æ–≤—å—è.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –Ø —Ä–∞–∑–±–∏—Ä–∞—é –∑–∞–ø—Ä–æ—Å –∫–ª–∏–µ–Ω—Ç–∞ —Å —Ä–∞–∑–Ω—ã—Ö —Å—Ç–æ—Ä–æ–Ω, —á—Ç–æ –ø–æ–º–æ–≥–∞–µ—Ç —É–≤–∏–¥–µ—Ç—å –ø–æ–ª–Ω—É—é –∫–∞—Ä—Ç–∏–Ω—É –∫–æ–Ω–∫—Ä–µ—Ç–Ω–æ–π —Å–∏—Ç—É–∞—Ü–∏–∏ –∏ –µ–µ –ø—Ä–∏—á–∏–Ω—ã. –Ø –∏—Å–ø–æ–ª—å–∑—É—é —Ä–∞–∑–ª–∏—á–Ω—ã–µ –ø–æ–¥—Ö–æ–¥—ã –∏ –∏–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç—ã, —á—Ç–æ–±—ã –ø–æ–º–æ—á—å –ø—Ä–æ—Ä–∞–±–æ—Ç–∞—Ç—å —ç—Ç–∏ –ø—Ä–∏—á–∏–Ω—ã –∏ –ø—Ä–∏–¥—Ç–∏ –∫ —Ä–µ—à–µ–Ω–∏—é –∑–∞–ø—Ä–æ—Å–∞ –∫–ª–∏–µ–Ω—Ç–∞.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –û–¥–Ω–∞ –∏–∑ –º–æ–∏—Ö –≥–ª–∞–≤–Ω—ã—Ö –∑–∞–¥–∞—á - –æ—Å–æ–∑–Ω–∞–Ω–∏–µ –∫–ª–∏–µ–Ω—Ç–æ–º —Ç–æ–≥–æ, 
    <strong
      className="css-1081t4c"
    >
      —á—Ç–æ, –∫–∞–∫ –∏ –ø–æ—á–µ–º—É –Ω—É–∂–Ω–æ –¥–µ–ª–∞—Ç—å
    </strong>
    , —á—Ç–æ–±—ã –¥–æ—Å—Ç–∏—á—å –∂–µ–ª–∞–µ–º–æ–≥–æ —Ä–µ–∑—É–ª—å—Ç–∞—Ç–∞.
  </p>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardInfo MuiAlert-standard css-12mvehg-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="InfoOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-1pxa9xg-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        –í–µ–¥—É—Ç—Å—è —Ä–∞–±–æ—Ç—ã
      </div>
      –í —Å–≤—è–∑–∏ —Å —Ç–µ–º, —á—Ç–æ –Ω–∞ —Å–∞–π—Ç–µ –≤–µ–¥—É—Ç—Å—è –∞–∫—Ç–∏–≤–Ω—ã–µ —Ä–∞–±–æ—Ç—ã, —á—Ç–æ-—Ç–æ –º–æ–∂–µ—Ç –æ—Ç—Å—É—Ç—Å—Ç–≤–æ–≤–∞—Ç—å, —Ä–∞–±–æ—Ç–∞—Ç—å –Ω–µ —Å—Ç–∞–±–∏–ª—å–Ω–æ –∏–ª–∏ –≤–æ–≤—Å–µ –Ω–µ —Ä–∞–±–æ—Ç–∞—Ç—å (–º–∞–ª–æ –≤–µ—Ä–æ—è–Ω–æ, –Ω–æ –≤—Å–µ-—Ç–∞–∫–∏ –≤–æ–∑–º–æ–∂–Ω–æ). –°–ø–∞—Å–∏–±–æ –∑–∞ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
    </div>
  </div>,
]
`;
